### PR TITLE
Linear range action rao tool

### PR DESCRIPTION
**Pull request by alexandrem-rte**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
New feature: it is now possible to launch a linear range action rao with itools.

**What is the current behavior?** *(You can also link to an open issue here)*
It was possible to launch only a search tree rao with itools.

**What is the new behavior (if this is a feature change)?**
Now it is possible to launch a search tree rao AND a linear range action rao with itools. It is possible to specify which rao implementation should be used, using a configuration file.

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*
Search tree rao command in itools has been removed

**Other information**:
Currently both RAOs return a NullPointerException, it seems to be normal as their development is still undergoing. Therefore the tests of this PR are not relevant in the long term and will have to change as soon as the RAOs return a result.
(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/RaoTool.java [hunk 1]
```diff
@@ -112,20 +109,23 @@ public void run(CommandLine line, ToolRunningContext context) throws Exception {
...
+        context.getOutputStream().println("Loading RAO parameters");
         RaoParameters raoParameters = RaoParameters.load(PlatformConfig.defaultConfig());
 
         //Run
         ComputationManager computationManager = context.getLongTimeExecutionComputationManager();
-        CompletableFuture<RaoComputationResult> raoComputationResult = new SearchTreeRao().run(network, crac, currentState, computationManager, raoParameters);
-        SearchTreeRaoResult searchTreeRaoResult = raoComputationResult.get().getExtension(SearchTreeRaoResult.class);
+        context.getOutputStream().println("Running Rao computation");
+        RaoComputationResult raoComputationResult = Rao.run(network, crac, currentState, computationManager, raoParameters);
+        RaoComputationResult raoResult = raoComputationResult.getExtension(RaoComputationResult.class);
```
Comment (baptsegu) [inline ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/RaoTool.java]: We should return directly the raoComputationResult. If you take only `raoComputationResult.getExtension(RaoComputationResult.class)` I'm afraid we might loose extension related to the different Rao implementations

Comment (alexandrem-rte) [reply ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/RaoTool.java]: corrected

File: ra-optimisation/rao-api/pom.xml [hunk 2]
```diff
@@ -40,6 +40,11 @@
             <artifactId>farao-ra-optimisation-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.farao-community.farao</groupId>
+            <artifactId>farao-crac-io-api</artifactId>
+            <version>1.3.0-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (alexandrem-rte) [reply ra-optimisation/rao-api/pom.xml]: ok

File: ra-optimisation/rao-api/pom.xml [hunk 3]
```diff
@@ -40,6 +40,11 @@
...
+        </dependency>
```
[the remark this answers is not in the document]
Comment (alexandrem-rte) [reply ra-optimisation/rao-api/pom.xml]: corrected

**Pull request merged**